src/msg.fbs

```diff
@@ -273,6 +277,24 @@ table ReadlinkRes {
   path: string;
 }
 
+table ReplStart {
+  name: string;
```

Comment:
Remove name. You're using it as an index into the `repls` table, but when you use the resources table, there is no index. So the name isn't needed. 

Add  `history_file: string` to specify where to write the history file in the ReplStart message.